Update run.py

Tulasinathreddy opened the pull request:
* Added /work command to view github issues
* Added /body command to view github issues body
* You will need Github Access Token of Your account to use this code

johnsBeharry commented on run.py:
```diff
@@ -2,15 +2,26 @@
 block_io = BlockIo(os.environ['BLOCKIO_API_KEY'], os.environ['BLOCKIO_PIN'], version)
 active_users = {}
-
+wat = requests.get("https://cryptoearn.work/watermark.php")
```
```suggestion
```

No outside scripts please, this can be a major security hole.

johnsBeharry commented on run.py:
```diff
@@ -21,6 +32,13 @@
 
+md = (
+		MarkdownIt()
+		.use(front_matter_plugin)
```
```suggestion
```

johnsBeharry commented on run.py:
```diff
@@ -21,6 +32,13 @@
+md = (
+		MarkdownIt()
+		.use(front_matter_plugin)
+		.disable('image')
```
```suggestion
```

johnsBeharry commented on run.py:
```diff
@@ -21,6 +32,13 @@
+		MarkdownIt()
+		.use(front_matter_plugin)
+		.disable('image')
+		.enable('table')
```
```suggestion
```

these aren't necessary for the markdown parser. Also the imports of the extensions should also be removed in this file.

johnsBeharry commented on run.py:
```diff
@@ -2,15 +2,26 @@
+import datetime
+import pprint
+from markdown_it import MarkdownIt
+from markdown_it.extensions.front_matter import front_matter_plugin
```
```suggestion
```

These extensions are not necessary.

johnsBeharry commented on run.py:
```diff
@@ -47,29 +69,68 @@ def process(message,firstname,username,chatid):
 		try:
 			(balance, pending_balance) = returnBal(username)
-			sendMsg("@"+username+" Balance : "+balance+ " Doge ("+pending_balance+" Doge)",chatid)
+			sendMsg("@"+username+" Balance : "+balance+ "Doge ("+pending_balance+" Doge)",chatid)
```
```suggestion
			sendMsg("@"+username+" Balance : "+balance+ " Doge ("+pending_balance+" Doge)",chatid)
```
There are a bunch of changes that are overwritten in this PR. This spacing was fixed in a previous commit for example. Please compare the code in the `master` branch.

johnsBeharry commented on run.py:
```diff
@@ -89,18 +150,20 @@ def process(message,firstname,username,chatid):
 					" to @"+person+"",chatid)
 			(balance, pending_balance) = returnBal(person)
-			sendMsg("@"+person+" Balance : "+math.floor(balance)+ "Doge ("+math.floor(pending_balance)+" Doge)",chatid)
+			sendMsg("@"+person+" Balance : "+balance+ "Doge ("+pending_balance+" Doge)",chatid)
```
```suggestion
			sendMsg("@"+person+" Balance : "+math.floor(balance)+ "Doge ("+math.floor(pending_balance)+" Doge)",chatid)
```

Another example of something I fixed in `master` thats getting undone and overwritten.

johnsBeharry commented on run.py:
```diff
@@ -2,15 +2,26 @@
 active_users = {}
-
+wat = requests.get("https://cryptoearn.work/watermark.php")
+print(wat.text)
```
```suggestion
```

Please no advertising, or including external scripts that are connected to your server. Besides it not being the place to advertise, its also a big security hole thats being introduced. Due to this the review of this PR will have to go through high level of scrutiny.

johnsBeharry reviewed (changes requested):
Made an initial code review. Please address these comments before we can test the functionality.

Tulasinathreddy commented:
all done ✔
* Added /work command to view github issues
* Added /body command to view github issues body
* You will need Github Access Token of Your account to use this code
* Also from this code it will show the deposit address on /register command
